recipe/meta.yaml

```diff
@@ -23,21 +24,21 @@ requirements:
   host:
     - cymem >=2.0.2,<2.1.0
     - cython >=0.25,<3.0
-    - cython-blis >=0.7.8,<0.8.0
+    - cython-blis >=1.0.0,<1.1.0
     - murmurhash >=1.0.2,<1.1.0
-    - numpy {{ numpy }}
+    - numpy >=2.0.0,<2.1.0
```

Comment:
just curious what the strategy for updating the cbc.yaml numpy pinning is? i.e. when do we do it? I'm wondering if it's better to have this pinning in the local cbc.yaml.

Reply:
Our base cbc.yaml pins numpy to 1.2X, but `spacy` and `thinc` are the first packages that dropped numpy <2 support. 
I would say a local cbc.yaml with `numpy 2.0` won't work:
```
numpy:
  - 2.0
  - 2.0
  - 2.0
  - 2.0
  - 2.0
```
```
because of an error:
 /Users/sssss/miniconda3/conda-bld/thinc_1728916870059/_build_env/bin/../include/c++/v1/stdio.h:107:15: fatal error: 'stdio.h' file not found
  #include_next <stdio.h>
                ^~~~~~~~~
```